example/src/main/java/com/stripe/example/activity/PaymentAuthActivity.kt

```diff
@@ -91,9 +91,20 @@ class PaymentAuthActivity : AppCompatActivity() {
         viewBinding.setupButton.setOnClickListener { createSetupIntent() }
     }
 
+    private val account: String?
+        get() {
+            val connectAccount = viewBinding.stripeAccount.text.toString()
```

Comment:
Instead of modifying this example, it might make sense to have a Connect integration example activity?

Reply:
Yeah, that's fair. Maybe I'll just remove the changes from these PR's and create a new PR to add an example activity?

Reply:
Yeah let's do that

Reply:
done!